.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -0,0 +1,39 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on PR
+'on': pull_request
+jobs:
+  build_and_preview:
+    if: '${{ github.event.pull_request.head.repo.full_name == github.repository }}'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node env
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14
+      
+      - name: Cache NPM Dependencies
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}    
+
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+      
+      - name: Generate
+        run: yarn run generate
+
+      - name: Run linter
+        run: yarn lint 
```

Comment:
it does seem kind of silly ;) - I feel this was auto generated by nuxt or something else a while ago. It could be useful if one looks at the logs, but in reality I should just get husky pre-commit hooks at some point. I'll remove